dirsrvtests/tests/suites/healthcheck/health_config_test.py

```diff
@@ -378,6 +380,7 @@ def test_healthcheck_notes_unindexed_search(topology_st, setup_ldif):
     log.info('Use filters to reproduce "notes=A" in access log')
     accounts = Accounts(standalone, DEFAULT_SUFFIX)
     accounts.filter('(uid=test*)')
+    sleep(1)
```

Comment:
Looks good but unsure it is always sufficient.  What do you think about a restart, would it be overkill ?

Reply:
I admittedly can't say for sure. I'll keep an eye on future regression tests in 389-ds-base branches under SE coverage and add a restart if the race condition continues.